.ci/watchdog/Main.py

```diff
@@ -106,6 +116,10 @@ def main(args):
 
     parser.add_argument('--quiet', help="Quiet mode - doesn\'t send message to slack channel.",
                         action='store_true', required=False)
+    parser.add_argument('--slack-enabled', type=int, help='Enable watchdog on Slack',
+                        default=0, action='store', required=False)
```

Comment:
The more canonical way would be using store_true action. You can define that parameter with `action="store_true"`, which will allow You to use it without explicit value, like this:
```
python Main.py --slack-enabled
```